.github/workflows/build-push-image-main.yml

```diff
@@ -0,0 +1,36 @@
+name: Tutor Build Action
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.main'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Get current time
```

Comment:
Is this package safe to use? Does it have any support? Did you try to create a function instead of using an external package? @sergivalero20

Reply:
Good question @Squirrel18 
I was find two alternatives to do that, but one of them didn't give me expected result, and the other one was deprecated. So, I decided to take this option. But, let me find a better solution or see if this solution is safe.